Apps/WebClient/src/NewClientApp/src/components/private/timeline/entry/BcCancerScreeningResultTimelineComponent.vue

```diff
@@ -0,0 +1,110 @@
+<script setup lang="ts">
+import saveAs from "file-saver";
+import { computed, ref } from "vue";
+
+import HgButtonComponent from "@/components/common/HgButtonComponent.vue";
+import MessageModalComponent from "@/components/common/MessageModalComponent.vue";
+import TimelineEntryComponent from "@/components/private/timeline/TimelineEntryComponent.vue";
+import { EntryType, entryTypeMap } from "@/constants/entryType";
+import { container } from "@/ioc/container";
+import { SERVICE_IDENTIFIER } from "@/ioc/identifier";
+import { PatientDataFile } from "@/models/patientDataResponse";
+import BcCancerScreeningResultTimelineEntry from "@/models/timeline/BcCancerScreeningResultTimelineEntry";
+import { ILogger } from "@/services/interfaces";
+import { usePatientDataStore } from "@/stores/patientData";
+import SnowPlow from "@/utility/snowPlow";
+
+interface Props {
+    hdid: string;
+    entry: BcCancerScreeningResultTimelineEntry;
```

Comment:
BcCancerScreeningResultTimelineComponent name is fine but are you about BcCancerScreeningResultTimelineEntry?  See comment in BcCancerScreeningResultTimelineEntry.

Reply:
Hi thanks for the feedback, I'm experiencing difficulty between the transition from API to front end which has too many mixes with `BcCancerScreening` vs `CancerScreening`. This just adds a lot of cognitive weight to carry around when working with the component.

Removing all references to `BcCancerScreening` from code (properties, component names etc.) and only leave it for values and display should reduce the complexity enough. I feel this should be alright. Thanks for the insight I'll add a task to refine this. The opposite would be enforce `BcCancerScreening` for all values including API responses. I'll see which way I go.